cpp/src/uuid.cpp

```diff
@@ -38,20 +43,17 @@
 namespace proton {
 
 namespace {
-
-
-// Seed the random number generated once at startup.
-struct seed {
-    seed() {
-        // A hash of time and PID, time alone is a bad seed as programs started
-        // within the same second will get the same seed.
-        unsigned long secs = time(0);
-        unsigned long pid = GETPID();
-        std::srand(((secs*181)*((pid-83)*359))%104729);
-    }
-} seed_;
-
-}
+// A hash of time, PID and random_device, time alone is a bad seed as programs
+// started within the same second will get the same seed.
```

Comment:
I think this comment maybe now incorrect as std::chrono::system_clock::now() isn't in seconds any more (is it?)

Reply:
Yeah, my bad! It was in nanoseconds. Updated the comment.